portability-transfer/src/main/java/org/datatransferproject/transfer/JobProcessor.java

```diff
@@ -46,12 +49,18 @@
   private final JobStore store;
   private final ObjectMapper objectMapper;
   private final InMemoryDataCopier copier;
+  private final Set<AuthDataDecryptService> decryptServices;
 
   @Inject
-  JobProcessor(JobStore store, ObjectMapper objectMapper, InMemoryDataCopier copier) {
+  JobProcessor(
+      JobStore store,
+      ObjectMapper objectMapper,
+      InMemoryDataCopier copier,
+      Set<AuthDataDecryptService> decryptServices) {
```

Comment:
For a future change, what do you think about making the backend determine which scheme to accept? I think it would be more secure than allowing the frontend request to specify it and allow cleartext.

Am I understanding it correctly that the backend defines a set of allowed schemes, and the frontend specifies which one (of the allowed) on a per-request basis?

Reply:
Actually, scratch that, just realized backend can prevent cleartext from being allowed